### PR TITLE
Making sure that the request states are not accidentally changed when user is admin

### LINKED ISSUE #278: The update  admin action web form should mention that it does not change state in pkdbb
Changing a status in
https://admin.fedoraproject.org/pkgdb/admin/action/1669/status
does not process the request but only changes the state of the request. This should be stated in the web page to make sure that request states are not accidently changed.

### PR DESCRIPTION
**Pull request by trishnaguha**
I am not able to see the issue since I was unable to make myself admin in my local instance, let me know if this fixes #278 
Thanks :)

### CODE REVIEW

**Comment by pypingou** [comment]
I'm not sure I agree with this change, it's a process that is blocked, not a state. Don't you think?

**Comment by pypingou** [comment]
I don't think this fixes the ticket #278, that ticket is asking to inform the admins that updating the status of a request does not actually process the request (ie: marking a branch request as ``approved`` does not actually create the new branch). The proper way to process requests is via the ``pkgdb-admin`` CLI tool.

File: pkgdb2/templates/actions_update.html [hunk 1]
```diff
@@ -35,7 +35,12 @@
 <form action="{{ url_for('.admin_action_edit_status', action_id=action_id) }}"
     method="post">
 {% endif %}
-
+{% if is_admin %}
```
Comment (trishnaguha) [inline pkgdb2/templates/actions_update.html]: @pypingou  Do I also have to check if fas user is in admin group? or only this will work?

Comment (pypingou) [inline pkgdb2/templates/actions_update.html]: It is sufficient, if the variable is set/given to this template :)

**Comment by ralphbean** [comment]
@pypingou, I think @trishnaguha made the update in response to your request.  Does it look okay now?

**Comment by pypingou** [comment]
Yes looks good and I checked, ``is_admin`` is defined so it'll work

**Pull request merged**